tests/glyphdata_test.py

```diff
@@ -44,6 +70,10 @@ def test_unicode(self):
         self.assertIsNone(uni("s_t"))  # no 'unicode' in GlyphsData
         self.assertEqual(uni("Gcommaaccent"), "Ģ")
         self.assertEqual(uni("o_f_f_i.foo"), "offi")
+        self.assertEqual(uni("brevecomb"), "̆")
```

Comment:
maybe use the `\u0123` escape instead of a literal unicode string here, as it's not really legible